spring-cloud-stream-binder-kinesis-core/pom.xml

```diff
@@ -26,6 +26,11 @@
 			<artifactId>spring-boot-configuration-processor</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-stream-binder-test</artifactId>
```

Comment:
Any particular reason to use this dependency?
I don't see any explanation in the commit messages, too.

Thanks